chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/statistics/NonconformityTypeStatisticsDAOImpl.java

```diff
@@ -0,0 +1,58 @@
+package gov.healthit.chpl.dao.statistics;
+
+import gov.healthit.chpl.dao.impl.BaseDAOImpl;
+import gov.healthit.chpl.dto.NonconformityTypeStatisticsDTO;
+import gov.healthit.chpl.entity.surveillance.NonconformityTypeStatisticsEntity;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Query;
+
+import org.springframework.stereotype.Repository;
+
+@Repository("nonconformityTypeStatisticsDAO")
+public class NonconformityTypeStatisticsDAOImpl extends BaseDAOImpl implements NonconformityTypeStatisticsDAO {
+
+	@Override
+	public List<NonconformityTypeStatisticsDTO> getAllNonconformityStatistics() {
+		String hql = "FROM NonconformityTypeStatisticsEntity WHERE deleted = false";
```

Comment:
More indentation things? Do you have Eclipse set up to use spaces instead of tabs in all cases?

Reply:
For some reason these errors are not showing up in my eclipse but im going to look at my eclipse style settings